scripts/scrape.py

```diff
@@ -0,0 +1,13 @@
+import os
+import time
+
+cities = [
+    "bangalore",
+    "pune",
+    "andhra"
+]
+
+while True:
+    for cmd in [f"python {city}.py" for city in cities]:
+        os.system(cmd)
```

Comment:
This looks good! Could you just add a line to log 'waiting for another hour to update...' or something similar?